### PR TITLE
YouTube-page-replica

### PR DESCRIPTION
**Pull request by Victorgonbu**
We have created a Youtube page replica using basic HTML and CSS.
Things we implemented:
-Header:
contains the youtube logo and some icons as embedded images as well as a search bar and sign in button.
-Main-content:
-added a warning label
-added video section
-added recommended videos section
-added user uploader details
-added video description

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,62 +5,160 @@
...
+                <figure><img src="./Images/thumbnail7.png" alt="
+                    Eat in one bite challenge | Tacos, Burritos (People Vs)"></figure>
+                <div>
+                    <p>Eat in one bite challenge | Tacos, Burritos (People Vs)</p>
+                    <h6>balerum <br>1,000 000 Views</h6>
+                </div>
+            </div>
+        </section>
+
+    </div>
```
Comment (lucasmazo32) [inline index.html]: These images are not properly loading to the page. Kindly check and fix the errors in here.

Here is a screenshot from your webpage.

image

File: index.html [hunk 2]
```diff
@@ -5,62 +5,160 @@
...
+                </div>
+
+            </div>
+
+        </div>
+
+        <div class="flexbox gray_border_top">
+            <img class="small_icons_square" src="./Images/add_icon.PNG" alt="add icon">
+            <p class="aling">Add to</p>
+            <img class="small_icons" src="./Images/share_icon1.png" alt="share icon">
```
Comment (lucasmazo32) [inline index.html]: The share icon is not loading as well. Kindly check what is happening.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good.

For the project to get approved kindly fix the error with some images loaded. I left a screenshot in the comments for you to see what is happening.

Last but not least, please do not close this Pull Request 🙏 . You can make changes and afterward leave a comment. This will help other TSE to understand what is missing from your project.

### Happy coding! :computer:

## Lucas Mazo

File: index.html (hunk 2 above)
Comment (Victorgonbu) [reply index.html]: ezgif com-video-to-gif

Hi Lucas, thanks for the code review, we have made the changes required!

File: index.html (hunk 1 above)
Comment (Victorgonbu) [reply index.html]: ezgif com-video-to-gif

the required changes are done!

**Review by serdg0 — CHANGES_REQUESTED** [review]
Hello @Shaqri and @balerum03 
Good job, but still there is a thing to improve it and we can approve it.

- [x] The share icon still is not visible. HINT: It's a typo.

Fix this issue and submit for another review.

You've done a great job so far :smile: 

Happy Coding! :computer: :nerd_face: 

**Sergio Diaz Gonzalez**

File: index.html [hunk 3]
```diff
@@ -140,7 +140,7 @@ <h3>
         <div class="flexbox gray_border_top">
             <img class="small_icons_square" src="./Images/add_icon.PNG" alt="add icon">
             <p class="aling">Add to</p>
-            <img class="small_icons" src="./Images/share_icon1.PNG" alt="share icon">
+            <img class="small_icons" src="./Images/share_icon.PNG" alt="share icon">
```
Comment (lucasmazo32) [inline index.html:143]: Hello team, the typo the last code reviewer was talking about is that you saved the icon as Share_icon.PNG, not share_icon.PNG.

**Review by lucasmazo32 — APPROVED** [review]
# Project approved :rocket:

Hello Team!
I want to give you advice for future projects.

The use of capital letters is not recommended in most cases when you are dealing with files. My advice is to use kebab-case for your images (names are separated by ```-``` and you only use lower case). Using capital letters can lead to mistakes like the one you were having with the display on some images for this project.

If you have any questions about this topic, you can reach me on slack! (lucasm32). 

Before merging please fix the typo that I pointed out in the comment!

### Happy coding! :computer:

## Lucas Mazo

File: index.html (hunk 3 above)
Comment (Victorgonbu) [reply index.html:143]: Hi Lucas, Thanks for everything! I was so blind, I couldn´t see what the problem was.

**Pull request merged**